packages/client/src/consts/environments.ts

```diff
@@ -1,17 +1,14 @@
 /**
  * @internal
- * @privateRemarks DO NOT export constructor, just the type and variables defined with it
  */
-class Environment {
```

Comment:
the constructor is not exported from the package, but it is used internally in `buildTestEnvironment`